framework/projects/Lang/compile-errors/test-65-65.diff

```diff
@@ -0,0 +1,16 @@
+index 38d578aba..a0ae9fc73 100644
+--- a/src/java/org/apache/commons/lang/enum/ValuedEnum.java
++++ b/src/java/org/apache/commons/lang/enum/ValuedEnum.java
+@@ -140,9 +140,9 @@ public abstract class ValuedEnum extends Enum {
+         }
+         List list = Enum.getEnumList(enumClass);
+         for (Iterator it = list.iterator(); it.hasNext();) {
+-            ValuedEnum enum = (ValuedEnum) it.next();
+-            if (enum.getValue() == value) {
+-                return.oldenum;
```

Comment:
This seems wrong (incorrect syntax before and after applying the patch) -- possibly an artifact of the enum -> oldenum replacement?

Reply:
The code is wrong before applying the patch due to the global enum->oldenum replacement.  This patch corrects the syntax errors by changing both 'enum' and 'oldenum' to 'enumer' and fixes the return statement by replacing '.' with ' '.

Reply:
My understanding of the regex is that it replaces `.enum;` with `.oldenum;`, which matches package declarations and import statements. This regex should *not* apply to the patched code: `return enum;` is missing a period for the regex to match. Am I missing something?

Reply:
bug in my sed script; needed to escape the escape: '\.'=>'\\.' will check in changes now